boost-maven/boost-runtimes/runtime-openliberty/src/it/java/boost/runtimes/utils/BoosterUtil.java

```diff
@@ -0,0 +1,22 @@
+package io.openliberty.boost.common.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.openliberty.boost.common.BoostException;
+import io.openliberty.boost.common.boosters.AbstractBoosterConfig;
+import io.openliberty.boost.common.boosters.JDBCBoosterConfig;
+
+public class BoosterUtil {
+
+    public static Map<String, String> createDependenciesWithBoosterAndVersion(Class<?> booster, String version)
+            throws BoostException {
+        Map<String, String> map = new HashMap<String, String>();
+        map.put(AbstractBoosterConfig.getCoordinates(booster), version);
+        return map;
+    }
+
+    public static Map<String, String> getJDBCDependency() throws BoostException {
+        return BoosterUtil.createDependenciesWithBoosterAndVersion(JDBCBoosterConfig.class, "0.1-SNAPSHOT");
+    }
+}
```

Comment:
Understand not wanting to pull out the UTs, but I wouldn't leave this layer for more than a short period...it's confusing and not doing anything useful.   If BoosterConfig(s) are common but these UTs are trying to do something Liberty-specific, let's not confusing things with this extra test layer.    

Until we decide what to do w/ BoosterConfig(s) we can leave it in I guess though.